src/tools/dart-compile.md

```diff
@@ -145,6 +145,19 @@ $ cd /tmp
 $ ./myapp
 ```
 
+#### Signing
+
+Executables created with `dart compile exe`
+support signing on macOS and Windows.
+
+For detailed documentation,
+see the platform documentation for those operating systems,
+such as the Windows [`SignTool.exe` documentation][],
+and the [Apple Code Signing guide][]
+
+[`SignTool.exe` documentation]: https://docs.microsoft.com/en-us/dotnet/framework/tools/signtool-exe
```

Comment:
Consider using the non-English link here so it can adapt to the user's choice.

```suggestion
[`SignTool.exe` documentation]: https://docs.microsoft.com/dotnet/framework/tools/signtool-exe
```